ibis/backends/dask/execution/util.py

```diff
@@ -31,7 +32,7 @@ def register_types_to_dispatcher(
             dispatcher.register(ibis_op, *types_to_register)(fn)
 
 
-def make_selected_obj(gs: SeriesGroupBy):
+def make_selected_obj(gs: SeriesGroupBy) -> Union[dd.DataFrame, dd.Series]:
```

Comment:
Since we have dd.DataFrame, maybe just use dd.SeriesGroupBy to be consistent

Reply:
`dd.SeriesGroupBy` doesn't exist. `DataFrameGroupBy` and `SeriesGroupBy` are in `dask.dataframe.groupby`

Reply:
Ok